lgsm/config-default/config-lgsm/mcserver/_default.cfg

```diff
@@ -89,7 +89,7 @@ engine="lwjgl2"
 ## Server Specific Directories
 systemdir="${serverfiles}"
 executabledir="${serverfiles}"
-executable="java -Xmx${javaram}M -jar ${serverfiles}/minecraft_server.jar nogui"
```

Comment:
`nogui` was required so would need to be in parameters instead.

Reply:
nogui is already in fn_parms, so the the result should be identical to this : added right after the executable.
My testing showed that linuxgsm would fail to find the executable with "nogui" added here, so any case, it's better removed, and that it also works perfectly fine without it.

Reply:
o yeah I missed that